cmd/utils/nodecmd/util.go

```diff
@@ -0,0 +1,237 @@
+}
+
+func parseGov(m map[string]interface{}, bytes []byte) error {
+	var b []byte
+	if err := rlp.DecodeBytes(bytes, &b); err == nil {
+		if err := json.Unmarshal(b, &m); err == nil {
+			return nil
+		} else {
+			return err
+		}
+	} else {
+		return err
+	}
+}
+
+func parseHeaderFile(headerFile string) (*types.Header, common.Hash, error) {
+	header := new(types.Header)
+	bytes, err := ioutil.ReadFile(headerFile)
+	if err != nil {
+		return nil, common.Hash{}, err
+	}
+	if err = json.Unmarshal(bytes, &header); err != nil {
+		return nil, common.Hash{}, err
+	}
+	var hash common.Hash
+	hasher := sha3.NewKeccak256()
+	rlp.Encode(hasher, types.IstanbulFilteredHeader(header, false))
+	hasher.Sum(hash[:0])
+	return header, hash, nil
+}
+
+func parseExtra(m map[string]interface{}, headerFile string) error {
+	header, hash, err := parseHeaderFile(headerFile)
+	if err != nil {
+		return err
+	}
+	istanbulExtra, err := types.ExtractIstanbulExtra(header)
+	if err != nil {
+		return err
+	}
+
+	validators := make([]string, len(istanbulExtra.Validators))
+	for idx, addr := range istanbulExtra.Validators {
+		validators[idx] = addr.String()
+	}
+	cSeals := make([]string, len(istanbulExtra.CommittedSeal))
+	for idx, cSeal := range istanbulExtra.CommittedSeal {
+		cSeals[idx] = hexutil.Encode(cSeal)
+	}
+
+	proposer, err := istanbul.GetSignatureAddress(hash.Bytes(), istanbulExtra.Seal)
+	if err != nil {
+		return err
+	}
+	m["validators"] = validators
+	m["seal"] = hexutil.Encode(istanbulExtra.Seal)
+	m["committedSeal"] = cSeals
+	m["validatorSize"] = len(validators)
+	m["committedSealSize"] = len(cSeals)
+	m["proposer"] = proposer.String()
+	return nil
+}
+
+func parseVote(m map[string]interface{}, bytes []byte) error {
+	vote := new(governance.GovernanceVote)
+	if err := rlp.DecodeBytes(bytes, &vote); err == nil {
+		m["validator"] = vote.Validator.String()
+		m["key"] = vote.Key
+		switch governance.GovernanceKeyMap[vote.Key] {
+		case params.GovernanceMode, params.MintingAmount, params.MinimumStake, params.Ratio:
+			m["value"] = string(vote.Value.([]uint8))
+		case params.GoverningNode:
+			m["value"] = common.BytesToAddress(vote.Value.([]uint8)).String()
+		case params.Epoch, params.CommitteeSize, params.UnitPrice, params.StakeUpdateInterval,
+			params.ProposerRefreshInterval, params.ConstTxGasHumanReadable, params.Policy, params.Timeout,
+			params.LowerBoundBaseFee, params.UpperBoundBaseFee, params.GasTarget, params.MaxBlockGasUsedForBaseFee, params.BaseFeeDenominator:
+			v := vote.Value.([]uint8)
+			v = append(make([]byte, 8-len(v)), v...)
+			m["value"] = binary.BigEndian.Uint64(v)
+		case params.UseGiniCoeff, params.DeferredTxFee:
+			v := vote.Value.([]uint8)
+			v = append(make([]byte, 8-len(v)), v...)
+			if binary.BigEndian.Uint64(v) != uint64(0) {
+				m["value"] = true
+			} else {
+				m["value"] = false
+			}
+		case params.AddValidator, params.RemoveValidator:
+			m["value"] = common.BytesToAddress(vote.Value.([]uint8)).String()
```

Comment:
`vote.Value` can contain an array of new/removed validator addresses.
https://github.com/klaytn/klaytn/blob/a7ac680ffc7267c2f51adc792992127648af54c3/governance/default.go#L573-L585